react/backstop/scripts/onBefore.js

```diff
@@ -0,0 +1,3 @@
+module.exports = async (page, scenario, vp) => {
+  // await require('./loadCookies')(page, scenario);
```

Comment:
I'm assuming this file is left here for future use?

Reply:
Yea, though I can't imagine we will ever need to add any cookies; maybe should just remove it.